BlakeDonn/fashion_wars
# Skin component bd

#### What's this PR do?  

- Divides skinDetails and previewSkin to two separate components to better split responsiblity
- Scrapes GW2 wiki for picture of armor set (currently only armor working)
- Adds wiki redirect for user when on skinDetails page
- Adds testing for unit for all new components
- Updates integration testing to include new user stories

#### Where should the reviewer start? 

- Start by running tests to get a good idea of the code

#### How should this be manually tested?  

- Run a local server test out the functionality
- Check for any inconsistencies in logic

#### Any background context you want to provide?  

- The scraping for the image is pretty barebones, may not implement armor and weapons in this scope

#### What are the relevant tickets?  

closes #22 
closes #24 
closes #25 
closes #26 
closes #27 
closes #28 

#### Screenshots (if appropriate) 

![Screen Shot 2020-11-09 at 3 32 10 PM](https://user-images.githubusercontent.com/65369275/98593149-bd3dfc80-22a0-11eb-85e9-6d67a8366870.png)

![Screen Shot 2020-11-09 at 3 32 24 PM](https://user-images.githubusercontent.com/65369275/98593175-c6c76480-22a0-11eb-8680-7031a0d1e2d6.png)

#### Questions:
